cmd/builder/internal/builder/main.go

```diff
@@ -86,7 +86,11 @@ func Compile(cfg Config) error {
 
 	cfg.Logger.Info("Compiling")
 	// #nosec G204
-	cmd := exec.Command(cfg.Distribution.Go, "build", "-ldflags=-s -w", "-trimpath", "-o", cfg.Distribution.Name)
+	args := []string{"build", "-ldflags=-s -w", "-trimpath", "-o", cfg.Distribution.Name}
+	if cfg.Distribution.BuildTags != "" {
+		args = append(args, "-tags", cfg.Distribution.BuildTags)
+	}
+	cmd := exec.Command(cfg.Distribution.Go, args...)
```

Comment:
Previously, the nosec line was right before the exec.Command, and it's not anymore. Move it here and it will skip the linter. 

That said, I'd be more comfortable if we were to sanitize both the argument that was originally causing G204 (cfg.Distribution.Name) and the new one (cfg.Distribution.BuildTags).

Reply:
I move the #nosec to the right place, lint is passed.
@TylerHelmuth @jpkrohling

Reply:
Are you able to sanitize it in any way, removing the need to skip the check?

Reply:
I agree with @jpkrohling.  It;s not the focus of the PR, but if we are gonna make changes to the Command we might as well try to go all the way and sanitize.

Reply:
I will fix this.

Reply:
After investigating all existing issues about the G204 error through [securego/gosec](https://github.com/securego/gosec) 

- https://github.com/securego/gosec/issues/204
- https://github.com/securego/gosec/issues/292
- https://github.com/securego/gosec/pull/339
- https://github.com/securego/gosec/issues/343
- https://github.com/securego/gosec/pull/677

even fix a bugs for G204 https://github.com/securego/gosec/pull/835 

I believe for the case of ocb, we should only escape the error by comments. This opinion is also approved by the author of gosec repo

https://github.com/securego/gosec/issues/831#issuecomment-1196292035

What do you think? @TylerHelmuth @jpkrohling

Reply:
I think the gosec line is already in the right place =D

Reply:
Tracking issue is already created #5781 @jpkrohling